README.md

```diff
@@ -1,36 +1,121 @@
-# Cairo VM in Typescript
+# TypeScript Cairo VM
+
+<div align="center">
+  <h1><code>TypeScript Cairo VM</code></h1>
+
+<strong>An implementation of the Cairo VM in TypeScript, focusing on
+education</strong>
+
+[Github](https://github.com/kkrt-labs/cairo-vm-ts)
+
+<sub>Built with 🥕 by <a href="https://twitter.com/KakarotZkEvm">KKRT
+Labs</a></sub>
+
+</div>
+
+> ⚠️ This project is in its genesis, undergoing fast-paced development. It is
+> not suitable for production yet. Expect frequent breaking changes.
 
 ## What is Cairo
 
-<!-- TODO: fill this section -->
+Cairo stands for CPU AIR "o" (like insp"o", conv"o", 🤔).
 
-Cairo stands for CPU AIR "o" (like insp"o", conv"o", 🤔). See
-[Cairo whitepaper](https://eprint.iacr.org/2021/1063) for more information.
+It is a framework (a machine, an assembly and a language) which allows writing
+provable programs without having to understand the underneath ZK-technology.
+
+A program written in Cairo (or Cairo Zero) is compiled and then executed on a
```

Comment:
*compiled to Cairo Assembly (CASM)

because you mention before machine + assembly + language, this makes it a bit clearer